interpreter.asm

```diff
@@ -110,9 +110,13 @@ interpret_loop
     rts
 
 interpret_tib
-    jsr	INTERPRET
+    jsr INTERPRET
     cpx #X_INIT+1
     bpl .on_stack_underflow
+    lda LATEST_MSB
+    sec
```

Comment:
Should I ignore the `sec` here?  Shouldn't make a difference.

Reply:
can't it be replaced by cmp?

Reply:
Oh yeah!

Reply:
This check isn't good enough, as HERE and LATEST can be on different pages with 1 byte unused.  I'll have to re-do it. (Also it's in the wrong place!)